### PR TITLE
Removes all the old gisf postback code

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jaycee** [comment]
No QA needed.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7304/

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -1287,18 +1287,12 @@ YUI.add('juju-gui', function(Y) {
         this.controllerAPI.userIsAuthenticated;
       const loginToController = controllerAPI.loginWithMacaroon.bind(
         controllerAPI, this.bakery);
-      const getDischargeToken = function() {
-        return window.localStorage.getItem('discharge-token');
-      };
-      const webhandler = new Y.juju.environments.web.WebHandler();
-      const charmstore = this.get('charmstore');
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: This is necessary for direct deploy, see line 1328

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7305/

**Comment by jaycee** [comment]
QA: Initially I thought this needed no QA, but given it whacked on DF and DD, you should probably make sure those both still work.

**Comment by frankban** [comment]
QA seems ok, dd works.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**